components/tabs/tabs.js

```diff
@@ -514,8 +519,22 @@ class Tabs extends LocalizeStaticMixin(ArrowKeysMixin(RtlMixin(LitElement))) {
 			.filter((node) => node.nodeType === Node.ELEMENT_NODE && node.role === 'tabpanel');
 	}
 
+	// IE11 only
+	_getTabInfoIndex(infos, id) {
+		for (let i = 0; i < infos.length; i++) {
+			if (infos[i].id === id) return i;
+		}
+		return -1;
+	}
+
 	_getTabInfo(id) {
-		return this._tabInfos.find((t) => t.id === id);
+		if (this._tabInfos.find) {
```

Comment:
We are not including a `find` polyfill though, so maybe that caused the issue?

Reply:
Yeah, `find` was the original issue, but then I ran into `findIndex` in core demo page while testing IE11.

Reply:
Ok I've updated to simply rely on the polyfill for `findIndex`, but to be safe, we should technically pull that polyfill into core so `d2l-tabs` can import it, since there are few things that are relying on tabs and I'm not certain they are all in BSI.